### DISCUSSION TITLE
Kuka 240-2 2000 series library

### DISCUSSION THREAD

**Discussion by mit6058** [visose/Robots#238] [Q&A] [answered]
Hello there, I've been trying to build the robot library for a KUKA 240-2 2000 series robot but I'm not able to find any solid documentation on the subject. Could someone help out with this?

**Comment by visose** [comment, marked as answer]
There is some information here: https://github.com/visose/Robots/wiki/Robot-libraries
You can find some posts about this if you search for `library` like this one: https://github.com/visose/Robots/discussions/140
